.github/workflows/deploy-to-lambda.yml

```diff
@@ -0,0 +1,30 @@
+name: deploy to lambda
+on:
+  push:
+    branches: ["release"]
+jobs:
+  deploy:
+    name: build and deploy lambda
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Run a multi-line script
```

Comment:
I think this name should be changed. I or someone else would have no idea of this step just by reading this title.

In this step, what you're doing is installing all dependencies and bundling them into a zip file. So I think we should keep the name that reflects what is being done in this step